RELEASE_NOTES.md

```diff
@@ -3,27 +3,42 @@
+- Quadlet now supports adding `tmpfs` filesystems through the Tmpfs key in .container files ([#17907](https://github.com/containers/podman/issues/17907)).
+- Quadlet now includes a `--version` option.
 - The `podman kube generate` and `podman kube play` commands now support ulimit annotations ([#16404](https://github.com/containers/podman/issues/16404)).
 - The `podman create`, `run`, `pod create`, and `pod clone` commands now support a new option, `--shm-size-systemd`, that allows limiting tmpfs sizes for systemd-specific mounts ([#17037](https://github.com/containers/podman/issues/17037)).
 - The `podman create` and `run` commands now support a new option, `--group-entry` which customizes the entry that is written to the `/etc/group` file within the container when the `--user` option is used ([#14965](https://github.com/containers/podman/issues/14965)).
+- The `podman create` and `podman run` commands now support a new option, `--security-opt label=nested`, which allows SELinux labeling within a confined container.
 - A new command, `podman machine os apply` has been added, which applies OS changes to a Podman machine, from an OCI image.
-- The `podman kube play` command now supports syctl options ([#16711](https://github.com/containers/podman/issues/16711)).
+- The `podman search` command now supports two new options: `--cert-dir` and `--creds`.
+- Defaults for the `--cgroup-config` option for `podman create` and `podman run` can now be set in `containers.conf`.
 - Podman now supports auto updates for containers running inside a pod ([#17181](https://github.com/containers/podman/issues/17181)).
-- Podman can now use a SQLite database as a backend for increased stability.
+- Podman can now use a SQLite database as a backend for increased stability. The default remains the old database, BoltDB. The database to use is selected through the `database_backend` field in `containers.conf`.
 
 ### Changes
 - Remote builds using the `podman build` command no longer allows `.containerignore` or `.dockerignore` files to be symlinks outside the build context.
 - The `podman system reset` command now clears build caches.
 - Quadlet now forbids specifying SELinux label types, including disabling selinux separation.
 - The `podman play kube` command now adds ctrName as an alias to the pod network ([#16544](https://github.com/containers/podman/issues/16544)).
 - The `podman kube generate` command no longer adds hostPort to the pod spec when generating service kinds.
-
+- Using a private cgroup namespace with systemd containers on a cgroups v1 system will explicitly error (this configuration has never worked) ([#17727](https://github.com/containers/podman/issues/17727)).
+- The `SYS_CHROOT` capability has been re-added to the default set of capabilities.
+- Listing large quantities of images with the `podman images` command has seen a significant performance improvement ([#17828](https://github.com/containers/podman/issues/17828)).
```

Comment:
In all cases, or when using sqllite?  If only sqllite, I'd specify that.

Reply:
All cases